lib/tokenize.cpp

```diff
@@ -9093,6 +9093,8 @@ void Tokenizer::simplifyNamespaceStd()
 
     for (const Token* tok = Token::findsimplematch(list.front(), "using namespace std ;"); tok; tok = tok->next()) {
         bool insert = false;
+        if (Token::Match(tok, "enum class|struct| %name%")) // #8091: Don't replace within enum definitions
```

Comment:
I would prefer that we don't write the ticket number in the code. Its pretty obvious that enums should be skipped. In the testcases I think it's ok with ticket numbers because the testcase is designed to test a specific ticket.